homeassistant/components/rest_command/__init__.py

```diff
@@ -118,13 +124,30 @@ async def async_service_handler(service: ServiceCall) -> None:
                 headers[hdrs.CONTENT_TYPE] = content_type
 
             try:
+                response: ClientResponse
                 async with getattr(websession, method)(
                     request_url,
                     data=payload,
                     auth=auth,
                     headers=headers,
                     timeout=timeout,
                 ) as response:
+                    return_data: ServiceResponse = (
+                        {
+                            "status": response.status,
+                            "headers": [
+                                {"key": item[0], "value": item[1]}
+                                for item in response.headers.items()
+                            ],
+                            "content": (await response.content.read()).decode(
```

Comment:
Should we automatically parse it as JSON if it's a JSON content type?

Reply:
I considered this and am happy to add it, but had erred on the side of not doing it because of the performance hit and it being easy to use `from_json` in a template for this purpose. But I don't feel strongly.

Reply:
My perspective is also that it might be best to return it as is to not restrict and limit scenarios.